sympy/functions/special/delta_functions.py

```diff
@@ -48,7 +51,7 @@ class DiracDelta(Function):
        Where ``x_i``-s are the roots of ``g``
     5) ``DiracDelta(-x) = DiracDelta(x)``
 
-    Derivatives of ``k``-th order of DiracDelta have the following property:
+    Derivatives of ``k``-th order of DiracDelta have the following properties:
 
     6) ``DiracDelta(x, k) = 0``, for all ``x != 0``
```

Comment:
And these three are

6. $\delta(x, k) = 0$ for all $x \neq 0$
7. $\delta(-x, k) = -\delta(x, k)$ for odd $k$
8. $\delta(-x, k) = \delta(x, k)$ for even $k$